editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -31,10 +31,25 @@
 #include "sprite_frames_editor_plugin.h"
 
 #include "core/io/resource_loader.h"
+#include "core/os/keyboard.h"
 #include "core/project_settings.h"
 #include "editor/editor_settings.h"
 #include "scene/3d/sprite_3d.h"
 
+void SpriteFramesEditor::_process_input(const Ref<InputEvent> &p_event, Node *p_hb) {
```

Comment:
`p_hb` is not really self-explanatory.

Reply:
renamed to p_from, so it should be more understandable :)